apps/market-maker-bancor/contracts/BancorMarketMaker.sol

```diff
@@ -353,12 +353,17 @@ contract BancorMarketMaker is EtherTokenConstant, IsContract, AragonApp {
         public view isInitialized
         returns (uint256 price)
     {
-        price = uint256(PPM).mul(
```

Comment:
Actually I'm not even sure we need the price in `PPM`. Because when we compute the exact amount of sell and buy return in the `clearMatching` function we divide by `PPM` again and got the exact same rounding error.

Let's say `pricePPM = 12345678` [so `priceNotPPM = 12`] and `totalBuySpend = 13`.

We then have with price in PPM:

`batch.totalBuyReturn = batch.totalBuySpend.mul(PPM).div(staticPricePPM); `
`batch.totalBuyReturn = 13 * PPM / 12345678`
`batch.totalBuyReturn = 13000000 / 12345678`
`batch.totalBuyReturn = 1`

And with priceNotPPM:
`batch.totalBuyReturn = batch.totalBuySpend.div(staticPrice); `
`batch.totalBuyReturn = 10 / 12`
`batch.totalBuyReturn = 1`

So in the end the rounding error is exactly the same. Right ?